src/styles/index.scss

```diff
@@ -1,3 +1,159 @@
-body {
+$background-size-h: 408px;
+$background-size-w: 200px;
+$blue-accent:  #00acdc;
+$button-h: 40px;
+$button-w: 166px;
+$border-radius: 5px;
+$star-size: 16px;
+$main-color:#060b35;
+$side-text-color: #616070;
+$light-color: #fff;
+$border-color: #f3f3f3;
+
+*,
+*::before,
+*::after {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+.card {
+  width: #{$background-size-w};
+  height: #{$background-size-h};
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  flex-shrink: 0;
+  border-radius: #{$border-radius};
+  border: 1px solid #{$border-color};
+  background: #{$light-color};
+  padding: 32px 16px 16px;
+}
+
+.image {
+  width: 160px;
+  height: 134px;
+  flex-shrink: 0;
+  margin-left: 3px;
+}
+
+.product {
+  flex-direction: column;
+  flex-shrink: 0;
+  color: #{$main-color};
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 18px;
+  margin-top: 36px;
+
+  &-code {
+    color: #{$side-text-color};
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    margin-top: 4px;
+  }
+}
+
+.block {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 16px;
+}
+
+.reviews {
+  flex-shrink: 0;
+  color: #{$main-color};
+  text-align: right;
+  font-size: 10px;
+  font-style: normal;
+  font-weight: 400;
+  display: flex;
+  flex-direction: column;
+  // justify-content: flex-end;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+  align-items: flex-start;
+
+}
+
+.price-text {
+  color: #{$side-text-color};
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 18px;
+}
+
+.price-num {
+  color: #{$main-color};
+  text-align: right;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 18px;
+}
+
+.buy-button {
+  width: #{$button-w};
+  height: #{$button-h};
+  flex-shrink: 0;
+  border-radius: #{$border-radius};
+  background: #{$blue-accent};
+  color: #{$light-color};
+  text-align: center;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 16px;
+  text-transform: uppercase;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 16px;
+
+  &:hover {
+    width: #{$button-w};
+    height: #{$button-h};
+    flex-shrink: 0;
+    border-radius: #{$border-radius};
+    border: 1px solid #{$blue-accent};
+    color: #{$blue-accent};
+    background-color: #{$light-color};
+  }
+}
```

Comment:
please take advantage of SCSS nesting, those all are part of one BEM block - card, please provide another file for card - [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) point 11